### PR TITLE
Modification social media rules

### PR DESCRIPTION
**Pull request by alexisljn**

### CODE REVIEW

File: models/forms/ManageProposalForm.php [hunk 1]
```diff
@@ -18,8 +18,8 @@ public function rules()
         return [
             [['title', 'content'], 'required'],
             ['relatedFile', 'file', 'extensions' =>
-                Util::UPLOADED_FILE_ALLOWED_EXTENSIONS,
-                'maxSize' => 52428800],
+                Util::UPLOADED_FILE_RULES,
```
Comment (EmileCalixte) [inline models/forms/ManageProposalForm.php]: The extensions of allowed files are now keys of this array. So You should use `array_keys(Util::UPLOADED_FILE_RULES)`

**Review by EmileCalixte — CHANGES_REQUESTED** [review]

File: models/forms/ManageProposalForm.php (hunk 1 above)
Comment (alexisljn) [reply models/forms/ManageProposalForm.php]: Oh you're so kind ! Thx for the advice :)

Comment (alexisljn) [reply models/forms/ManageProposalForm.php]: It's ARRAY NOT ARAY DUDE

Comment (EmileCalixte) [reply models/forms/ManageProposalForm.php]: I don't see what you mean

**Comment by alexisljn** [comment]
So you want to F* with me ?

**Review by EmileCalixte — APPROVED** [review]

**Review by GuenoleCHERUBIN — APPROVED** [review]

**Pull request merged**